plugins/azure-devops/src/components/PullRequestsPage/lib/PullRequestCard/PullRequestCardReviewer.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Avatar } from '@backstage/core-components';
+import React from 'react';
+import { Reviewer } from '@backstage/plugin-azure-devops-common';
+
+type PullRequestCardReviewerProps = {
+  reviewer: Reviewer;
+};
+
+export const PullRequestCardReviewer = ({
+  reviewer,
+}: PullRequestCardReviewerProps) => (
+  <Avatar
+    displayName={reviewer.displayName}
+    picture={reviewer.imageUrl}
+    customStyles={{
+      width: '2.5rem',
+      height: '2.5rem',
+      fontSize: '1rem',
+      border: '0.3rem solid silver',
```

Comment:
This will be used to display the reviewer vote status:
![image](https://user-images.githubusercontent.com/55280588/141462259-e58491be-fc3a-416b-a238-37b1b65fc9a0.png)

I am thinking of maybe doing this a bit differently but for now will leave it as it is. Once everything is wired up and working I definetely would like design feedback for things that could be improved.

Reply:
Ah, this makes more sense to me now, thanks for the extra context